client/plugin_install.go

```diff
@@ -52,6 +46,21 @@ func (cli *Client) PluginInstall(ctx context.Context, name string, options types
 		}
 	}
 
+	_, err = cli.tryPluginPull(ctx, query, privileges, options.RegistryAuth)
+	if err != nil {
+		ensureReaderClosed(resp)
```

Comment:
`resp` is already closed in L37. For `tryPluginPull`, we ignore `resp`. So there's no need to close in L51 and L55.